.github/workflows/test-aks.yaml

```diff
@@ -0,0 +1,369 @@
+            rad env switch ${{ matrix.env }}
+          elif [[ "${{ matrix.credential }}" == "aws" ]]; then
+            rad deploy ./samples/eshop/environments/aws.bicep -p awsAccountId=${{ secrets.AWS_ACCOUNT_ID }} -p awsRegion=${{ env.AWS_REGION }} -p eksClusterName=${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }}
+            rad env switch ${{ matrix.env }}
+          else
+            echo "Registering recipes for ${{ matrix.env }} environment..."
+            rad recipe register default -e ${{ matrix.env }} --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/rediscaches:latest --resource-type Applications.Datastores/redisCaches
+            rad recipe register default -e ${{ matrix.env }} --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/mongodatabases:latest --resource-type Applications.Datastores/mongoDatabases
+            rad recipe register default -e ${{ matrix.env }} --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/sqldatabases:latest --resource-type Applications.Datastores/sqlDatabases
+            rad recipe register default -e ${{ matrix.env }} --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/rabbitmqqueues:latest --resource-type Applications.Messaging/rabbitMQQueues
+          fi
+      - name: Configure cloud credentials
+        if: steps.gen-id.outputs.RUN_TEST == 'true' && ( matrix.credential == 'azure' || matrix.credential == 'aws')
+        run: |
+          if [[ "${{ matrix.credential }}" == "azure" ]]; then
+            rad env update ${{ matrix.env }} --azure-subscription-id ${{ secrets.AZURE_SANDBOX_SUBSCRIPTION_ID }} --azure-resource-group ${{ steps.gen-id.outputs.TEST_AZURE_RESOURCE_GROUP }}
+            rad credential register azure --client-id ${{ secrets.AZURE_SANDBOX_APP_ID }} --client-secret ${{ secrets.AZURE_SANDBOX_PASSWORD }} --tenant-id ${{ secrets.AZURE_SANDBOX_TENANT_ID }}
+          fi
+          if [[ "${{ matrix.credential }}" == "aws" ]]; then
+            rad env update ${{ matrix.env }} --aws-region ${{ env.AWS_REGION }} --aws-account-id ${{ secrets.AWS_ACCOUNT_ID }}
+            rad credential register aws --access-key-id ${{ secrets.AWS_ACCESS_KEY_ID }} --secret-access-key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          fi
+      # Deploy application and run tests
+      - name: Deploy app
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        id: deploy-app
+        run: rad deploy ${{ matrix.path }} ${{ matrix.deployArgs }} -e ${{ matrix.env }}
+      - name: Wait for all pods to be ready
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        id: wait-for-pods
+        run: |
+          namespace="${{ matrix.env }}-${{ matrix.app }}"
+          label="radapp.io/application=${{ matrix.app }}"
+          kubectl rollout status deployment -l $label -n $namespace --timeout=90s
```

Comment:
It would be good to remove this if we can. This is guarantee that Radius is already providing when you deploy the application.

Reply:
We have some instances where one of Pod is not available after the deployment: https://github.com/radius-project/samples/issues/1057. Investigating further @rynowak.